packages/dds/tree/src/events/fence.json

```diff
@@ -1,5 +1,5 @@
 {
   "tags": ["events"],
   "exports": ["index"],
-  "imports": []
```

Comment:
you changed a fence file and did not update the readme!

Reply:
Updated!